pro/src/repository/pcapi/pcapi.js

```diff
@@ -244,6 +244,10 @@ export const editEducationalStock = (stockId, stock) => {
   return client.patch(`/stocks/educational/${stockId}`, stock)
 }
 
+export const cancelEducationalBooking = offerId => {
+  return client.patch(`/offers/${offerId}/cancel_booking`)
```

Comment:
Je débarque complètement, mais je mettrais bien un "s" à "bookings", parce qu'ici on annule **toutes** les réservations de l'offre demandée.

```suggestion
  return client.patch(`/offers/${offerId}/cancel_bookings`)
```

... on pourrait au passage ajouter le même "s" côté backend : la route `pcapi.routes.pro.offers..cancel_educational_offer_booking()` et la fonction `pcapi.core.offers.api.cancel_educational_offer_booking()`

Reply:
@dbaty côté EAC, on ne peut avoir qu'un seul booking et qu'une seul stock...
d'ailleurs, si l'offre à plusieurs stocks, cette route lève une exeption

Reply:
Ah, ok, ça se défend, alors.